cmd/generate/generate.go

```diff
@@ -33,10 +33,11 @@ import (
 	routeapi "github.com/openshift/origin/pkg/route/api/v1"
 	templateapi "github.com/openshift/origin/pkg/template/api/v1"
 	userapi "github.com/openshift/origin/pkg/user/api/v1"
-	appsapi "k8s.io/kubernetes/pkg/apis/apps/v1alpha1"
 	resourceapi "k8s.io/kubernetes/pkg/api/resource"
 	rapi "k8s.io/kubernetes/pkg/api/unversioned"
 	kapi "k8s.io/kubernetes/pkg/api/v1"
+	appsapi "k8s.io/kubernetes/pkg/apis/apps/v1beta1"
+	batchapi "k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
```

Comment:
You have to use the versioned APIs, so v1, v1beta1, etc. I guess we should stick with v2alpha1 as that is the only version where CronJob exists. OK let's leave it as is. We need to handle multiple versions of APIs in model v2 (some time).